### PR TITLE
Table to better understand ZHA Radio Types & HW

### PR DESCRIPTION
**Pull request by rosscullen**
@frenck when you get a chance, could you review the proposed? From doing a ZHA Integrations install today (after seeing your livestream topic this week), I didn't know which radio type to pick for a Conbee II. Hopefully this table will help users with their install. 

It's hard to figure out which hardware is compatible with some of the radio types. Would you or someone  be able to validate and/or add in the other Radio Hardware to the table? Thank you!

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:
- This PR fixes or closes issue:

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by Adminiuga** [comment]
Please add Zigate and TI CC based radios. See #12220

**Review by klaasnicolaas — DISMISSED** [review]
```
/home/runner/work/home-assistant.io/home-assistant.io/source/_integrations/zha.markdown
  86:3  ✓ error  Incorrect usage of the term: “deconz”, use “deCONZ” instead  terminology
```

**Comment by Adminiuga** [comment]
hrm, in that context it is talking about "radio types" available in the drop down box when you configure ZHA integration config_entry. All those radio types are in lower case. 
So the question here: Do we document it "case insensitively" or should it be documented matching letter by letter.

**Comment by frenck** [comment]
They need to code blocked. Will add review suggestion to address the build.

File: source/_integrations/zha.markdown [hunk 1]
```diff
@@ -77,11 +77,18 @@ Use the plus button in the bottom right to add a new integration called **ZHA**.
...
+- USB Device Path - on a Linux system will be something like `/dev/ttyUSB0` or `/dev/ttyACM0`
 - Radio type - select device type `ezsp`, `deconz` or `xbee`
-- Submit
 
-The success dialog will appear or an error will be displayed in the popup. An error is likely if Home Assistant can't access the USB device or your device is not up to date (see troubleshooting).
+| Radio Type | Zigbee Radio Hardware |
+| ------------- | ------------- |
+| ezsp  | EmberZNet based radios, Telegesis ETRX357USB*** (using EmberZNet firmware)  |
+| deconz | ConBee, ConBee II |
+| xbee | Digi XBee Series 2, 2C, 3  |
```
Comment (frenck) [inline source/_integrations/zha.markdown]:
```suggestion
| `ezsp`  | EmberZNet based radios, Telegesis ETRX357USB*** (using EmberZNet firmware)  |
| `deconz` | ConBee, ConBee II |
| `xbee` | Digi XBee Series 2, 2C, 3  |
```

File: source/_integrations/zha.markdown [hunk 2]
```diff
@@ -138,7 +145,7 @@ Click on **ADD DEVICES** to start a scan for new devices.
 
 Reset your Zigbee devices according to the device instructions provided by the manufacturer (e.g.,  turn on/off lights up to 10 times, switches usually have a reset button/pin).
 
-## Troubleshooting
+# Troubleshooting
```
Comment (frenck) [inline source/_integrations/zha.markdown]: The H1 (first header) is reserved for the title only. Please use an H2 at a minimum.
```suggestion
## Troubleshooting
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by Gamester17** [comment]
@rosscullen, good point. I have also tried to make radio-type choice a little bit clearer in PR #12233

File: source/_integrations/zha.markdown (hunk 2 above)
Comment (rosscullen) [reply source/_integrations/zha.markdown]: Thanks @frenck @klaasnicolaas @Adminiuga for feedback. Can I double check, do you think the list of compatible Zigbee hardware covers them all? Thanks

**Comment by Gamester17** [comment]
Should also the default device port path be documented too?

ConBee I = /dev/ttyUSBx
ConBee II = /dev/ttyACMx
RaspBee I / II = /dev/ttyAMA0 or /dev/ttyS0

Texas Instruments CC2531 = /dev/ttyACM0

ZiGate USB-TTL / USB-DIN = /dev/ttyUSB0

**Comment by frenck** [comment]
@Gamester17 You can't state that it can differ on systems depending on how it is hooked up. Furthermore, I definitely do not recommend using those device paths, instead use the serial by id path, as they are stable.

**Review by rosscullen — COMMENTED** [review]
Changes commited. Please can someone approve? I will try and review the page in further detail at a later stage. Thanks

**Review by frenck — DISMISSED** [review]
Thanks, @rosscullen! 👍

**Review by frenck — APPROVED** [review]

**Pull request merged**